oas

```diff
@@ -1,506 +1,343 @@
+                        "example": "12:34:56:78:90:AB"
+                    },
+                    "data": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/SensorData"
+                        }
+                    }
+                },
+                "description": "Data model for the data response"
+            },
+            "Measurements": {
+                "title": "Measurements",
+                "type": "object",
+                "properties": {
+                    "temperature": {
+                        "type": "number",
+                        "description": "Measured temperature of the asset's temperature sensor in celsius degree",
+                        "example": 29.5
+                    },
+                    "humidity": {
+                        "type": "number",
+                        "description": "Measured humidity of the asset's humidity sensor in percentage",
+                        "example": 50.1
+                    },
+                    "pressure": {
+                        "type": "number",
+                        "description": "Measured pressure of the asset's pressure sensor in hectopascal",
+                        "example": 1020
+                    }
+                },
+                "description": "Data Model defining measurements for sensor data"
             },
-            "example": [
-              "optional",
-              "additional",
-              "details"
-            ]
-          }
+            "SensorData": {
+                "title": "SensorData",
+                "type": "object",
+                "properties": {
+                    "timestamp": {
+                        "type": "string",
+                        "description": "ISO-8601 timestamp of the data point",
+                        "format": "date-time",
+                        "example": "2022-10-18T13:34:29Z"
+                    },
+                    "coordinates": {
+                        "type": "array",
+                        "items": {},
+                        "description": "Coordinate of the point, in WGS84 reference system. [Longitude, Latitude]",
+                        "example": [
+                            25.312,
+                            48.578
+                        ]
+                    },
+                    "measurements": {
+                        "$ref": "#/components/schemas/Measurements"
+                    }
+                },
+                "description": "Data Model for asset's sensor information. Information are as follows: <br>\nTimestamp in ISO-8601 format <br>\nGPS location:<br>\n    <ol>\n    <li> Latitude </li>\n    <li> Longitude </li>\n    </ol>\nMeasurements:<br>\n    <ol>\n    <li> Temperature in Celcius </li>\n    <li> Humidity in Percentage </li>\n    <li> Pressure in Hectopascal </li>\n    </ol>"
```

Comment:
Use "required" array to indicate which properties MUST be present in all responses.
In this case, probably ["timestamp", "coordinates", "measurements"]